internal/kv/mem/mem_kv.go

```diff
@@ -97,6 +97,17 @@ func (kv *MemoryKV) LoadBytes(key string) ([]byte, error) {
 	return item.(memoryKVItem).value.ByteSlice(), nil
 }
 
+// Get return value if key exists, or return empty string
+func (kv *MemoryKV) Get(key string) string {
```

Comment:
why not use load?

Reply:
Load is weird that will return an error if not found value, it does not make sense for a KV interface